tests/phpunit/data/cross_origin_content.html

```diff
@@ -0,0 +1,21 @@
+<html amp="" lang="en">
+<head>
+  <meta charSet="utf-8" />
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+  <script async="" src="https://cdn.ampproject.org/v0.js"></script>
+  <script async="" src="--SITE_URL--"></script>
+  <script async="" src="https://cdn.ampproject.org/v0/amp-story-1.0.js" custom-element="amp-story"></script>
+  <link href="https://fonts.googleapis.com/css2?display=swap&#038;family=Roboto" rel="stylesheet" />
+  <link href="--SITE_URL--/site.css" rel="stylesheet" />
+</head>
+<body>
+<div>
```

Comment:
Could you add some `<a>` tags here as well to see how they are affected?

Reply:
Added.